src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+  </Side>
+  </SideBySide>
+
+# Infrastructure [#infrastructure]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="infrastructure" /> capability lets you monitor the health and performance of your service, including containers and cloud infrastructure. It covers tracking CPU, memory, network traffic, and disk use. It also includes over 400 on-host integrations for monitoring third-party apps.
+
+  Learn more about infrastructure [here](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic infrastructure"
+    alt="New Relic infrastructure"
+    src={NRInfraMain}
+  />
+  </Side>
+  </SideBySide>
+
+# Kubernetes and Pixie [#k8s-pixie]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Kubernetes"
+    alt="New Relic Kubernetes"
+    src={NRK8sMain}
+  />
+
+  </Side>
+  <Side>
+
+  The New Relic Kubernetes integration gives you full observability. The data it provides lets you monitor the health of your whole Kubernetes cluster and the performance of your environment. You can check individual pods or drill into specific services and applications. The agent collects telemetry data from your cluster. It uses several New Relic integrations, including Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
+
+  Learn more about Kubernetes [here](l/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration).
+
+  </Side>
+  </SideBySide>
+
+# Live archives [#live-archives]
+<SideBySide>
+  <Side>
+
+  Live archives store historical logs with other logs and telemetry data in the New Relic Database. This allows instant access to old logs and stops the need to reload, re-index logs, or move data between many locations or tiers for analysis.
+
+  Learn more about live archives [here](/docs/logs/get-started/log-live-archives).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLiveArchivesMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# Logs [#logs]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLogsMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic can help you manage large volumes of your log data. We provide tools to ingest only valuable logs to cut costs. We offer a unified UI to connect your <InlinePopover type="logs" /> to your services. We also offer tools that can help you sort through the clutter. 
+
+  Learn more about logs [here](/docs/logs/get-started/get-started-log-management/).
+
+  </Side>
+  </SideBySide>
+
+# Network [#network]
+<SideBySide>
+  <Side>
+
+  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology, letting you find performance problems in your environment. 
```

Comment:
```suggestion
  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology, letting you find performance problems in your environment. 
```
```suggestion
  With <InlinePopover type="networkMonitoring" />, you can analyze the performance of your routers, switches, and other networking devices. Network monitoring builds a visual network topology for you, so you can identify when a performance problem is occurring in the network layer.
```

Reply:
Great! I've changed the paragraph ;-)